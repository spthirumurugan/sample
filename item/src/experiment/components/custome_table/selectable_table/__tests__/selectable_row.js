import React from 'react';
import renderer from 'react-test-renderer';
import SelectableRow from '../selectable_row';

describe('components', () => {
  describe('<SelectableRow />', () => {
    it('should render', () => {
      const row = renderer.create(
        <SelectableRow
          cells={['prop1', 'prop2']}
          uIds={['1', '2', '3', '4']}
          id={'id_1'}
          checked
          onCheck={jest.fn()}
        />
      ).toJSON();
      expect(row).toMatchSnapshot();
    });
  });
});
