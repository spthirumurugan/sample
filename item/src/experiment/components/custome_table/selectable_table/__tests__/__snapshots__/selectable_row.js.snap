// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components <SelectableRow /> should render 1`] = `
<tr>
  <td>
    <div
      className="checkbox checkbox-green"
    >
      <input
        checked={true}
        id="checkbox_id_17l9p5j1dbgj0cexnow"
        onChange={[Function]}
        type="checkbox"
        value="id_1"
      />
      <label
        htmlFor="checkbox_id_17l9p5j1dbgj0cexnow"
      />
    </div>
  </td>
  <td>
    prop1
  </td>
  <td>
    prop2
  </td>
</tr>
`;
